.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,45 @@
+name: Integration Tests
+on:
+  push:
+    branches:
+      - master
+      - develop
+  pull_request:
+    branches:
+      - master
+      - develop
+
+env:
+  REACT_APP_NETWORK_URL: 'https://mainnet.infura.io/v3/4bf032f2d38a4ed6bb975b80d6340847'
+  APP_ID: 1
+
+jobs:
+  integration-tests:
+    name: Cypress
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 12
+
+      # Install deps only
+      - name: Cypress install deps
+        uses: cypress-io/github-action@v2
+        with:
+          runTests: false
+
+      - name: Install Cypress binary
+        if: failure()
```

Comment:
I didn't understand why this condition is needed. Is not counter-intuitive?

Reply:
My thought process is:
- cypress action fails because it cannot find the binary
- when that happens, the step will fail
- so, I split the cypress action into 2: install and run
- if the install step fails, it is very likely due to a missing binary, so, install it in this step
- on the last step, run the actual test

That being said, it did not fail as it was for the past 2 days and worked on the first try...